### DISCUSSION TITLE
Congratulations on 100 github stars!

### DISCUSSION THREAD

**Discussion by epogrebnyak** [thuliteio/doks#99] [General]
![изображение](https://user-images.githubusercontent.com/9265326/106472284-92d09980-64b3-11eb-94dc-ced6c50ead04.png)

Congratulations @h-enk and theme users on first 100 stars on Github! Looking forward to 100 * 100!

**Comment by h-enk** [comment]
Thanks @epogrebnyak, and thank you [all 100 stargazers](https://github.com/h-enk/doks/stargazers) for starring the Doks repo! It really motivates to push Doks even further — expect some interesting features coming! 

Also thanks @algolia for sponsoring the Doks documentation search, and @netlify for sponsoring the Doks hosting infrastructure.
